docs/build.gradle

```diff
@@ -1,45 +1,30 @@
-apply plugin: "com.github.node-gradle.node"
-
-node {
-    version = '14.16.0'
-    yarnVersion = '1.22.10'
-    npmVersion = "6.14.18"
-    download = true
+plugins {
+    id 'base'
 }
 
-task build(type: YarnTask, dependsOn: yarn_install) {
-
-    node {
-        version = '14.16.0'
-        yarnVersion = '1.22.10'
-        npmVersion = "6.14.18"
-    }
-
-    inputs.files(fileTree('src'))
-    inputs.files(fileTree('public'))
-    inputs.file('package.json')
-    outputs.dir('src/.vitepress/dist')
-
-    // tell gradle to apply the build cache
+task copyDistFiles(type: Copy) {
+    from "."
+    exclude 'build'
+    into "${project.buildDir}/resources/main/public_html/apps/${project.name}"
+    outputs.dir("${project.buildDir}/resources/main/public_html/apps/${project.name}")
     outputs.cacheIf { true }
-
-    //first install the workspace
-    args = ['run', 'build']
-
-    //remove the target folder with copy of all build materials
-    delete "${project.parent.buildDir}/resources/main/public_html/apps/${project.name}"
 }
 
-task copyDistFiles(type: Copy, dependsOn: buildDocs) {
-    from "src/.vitepress/dist"
-    inputs.dir("src/.vitepress/dist")
-
-    into "${project.parent.buildDir}/resources/main/public_html/apps/${project.name}"
-    outputs.dir("${project.parent.buildDir}/resources/main/public_html/apps/${project.name}")
-    outputs.cacheIf { true }
+task createJar(type: Zip, dependsOn: copyDistFiles) {
+    baseName 'docs'
```

Comment:
why is the baseName 'docs' ?

Reply:
this is part of the 'jar' that is created; is for internal use only as this jar is not published (we have a fatjar). The 'apps' jar uses 'apps' as base name so that is why I thought 'docs' to be appropriate. If I look to the other jars we probably should change base name for both of them to e.g. molgenis-emx2-docs.jar and molgenis-emx2-apps.jar respectively.